Naughtya/Projects/Features/CoreFeature/Sources/Presentation/Views/TodoItem/TodoItemView.swift

```diff
@@ -86,6 +93,34 @@ public struct TodoItemView: View {
         .frame(height: 40)
         .background(.white)
         .opacity(opacity)
+        .gesture(
+            DragGesture()
+                .onChanged {
+                    let itemLocation = absoluteRect.origin + $0.location - $0.startLocation
+                    if !isBeingDragged {
+                        dragDropDelegate.startToDrag(
+                            todo.entity,
+                            size: absoluteRect.size,
+                            itemLocation: itemLocation
+                        )
+                    } else {
+                        dragDropDelegate.drag(
+                            todo.entity,
+                            itemLocation: itemLocation
+                        )
+                    }
```

Comment:
```suggestion
                    if isBeingDragged {
                        dragDropDelegate.drag(
                            todo.entity,
                            itemLocation: itemLocation
                        )
                    } else {
                        dragDropDelegate.startToDrag(
                            todo.entity,
                            size: absoluteRect.size,
                            itemLocation: itemLocation
                        )
                    }
```

!를 없애고 반대로 넣으면 읽기 쉬울 것 같아요 ~

Reply:
로직이 항상 startToDrag -> drag 순서로 실행돼서 순서를 기준으로 작성했는데
이런 케이스에도 flag를 기준으로 작성하는 걸 추천하실까요?